src/views/Wrap/components/MigrateInputArea/MigrateInputArea.tsx

```diff
@@ -0,0 +1,99 @@
+import { t, Trans } from "@lingui/macro";
+import { Box, FormControl, Grid, MenuItem, Select, Typography } from "@material-ui/core";
+import { Input, PrimaryButton } from "@olympusdao/component-library";
+import { useState } from "react";
+import { TokenAllowanceGuard } from "src/components/TokenAllowanceGuard/TokenAllowanceGuard";
+import { MIGRATOR_ADDRESSES, WSOHM_ADDRESSES } from "src/constants/addresses";
+import { assert } from "src/helpers/types/assert";
+import { useWeb3Context } from "src/hooks";
+import { useBalance } from "src/hooks/useBalance";
+import { useTestableNetworks } from "src/hooks/useTestableNetworks";
+
+import { useMigrateWsohm } from "./hooks/useMigrateWsohm";
+
+export const MigrateInputArea = () => {
+  const networks = useTestableNetworks();
+  const { networkId } = useWeb3Context();
+
+  // Max balance stuff
+  assert(
+    networkId === networks.ARBITRUM || networkId === networks.AVALANCHE,
+    "Component should only be mounted when connected to Arbitrum or Avalanche",
+  );
+  const [amount, setAmount] = useState("");
+  const balance = useBalance(WSOHM_ADDRESSES)[networkId].data;
+  const setMax = () => balance && setAmount(balance.toAccurateString());
+
+  // Mutation stuff
+  const migrateMutation = useMigrateWsohm();
+  const handleSubmit = (event: React.FormEvent<WrapFormElement>) => {
+    event.preventDefault();
+    migrateMutation.mutate(amount);
+  };
+
+  return (
```

Comment:
something misaligned here 
<img width="877" alt="image" src="https://user-images.githubusercontent.com/80423742/159719671-03b2163a-5c98-4dfa-8059-cfc405835f42.png">

Reply:
the correct solution should be to bring the prompt `Enter an amount of wsOHM` into the input. That way it matches all of our other inputs (see staking & bonding)

Reply:
here's how the mainnet wrap input looks
<img width="805" alt="image" src="https://user-images.githubusercontent.com/80423742/159721985-c4825b67-a928-4957-9c6f-16d73e678414.png">

Reply:
I think this was due to an update to the input component in our component library?